src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -386,6 +379,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 			let elementContainingText = elements[range.startLineNumber - 1];
 			let mark = new Mark(elementContainingText);
 			mark.unmark({ acrossElements: true, className: 'rangeHighlight' });
+			mark.unmark({ acrossElements: true, className: 'rangespecifcHighlight' });
```

Comment:
is there a reason you're not capitalizing specific? `rangeSpecificHighlight`

Reply:
this was supposed to be removed, but yeah I think I just missed the casing there.